core/modules/loaders.rs

```diff
@@ -78,9 +80,7 @@ impl ModuleLoader for NoopModuleLoader {
     referrer: &str,
     _kind: ResolutionKind,
   ) -> Result<ModuleSpecifier, Error> {
-    Err(generic_error(
```

Comment:
Note that I'm intentionally changing `NoopModuleLoader` to allow for resolution but not loading -- this allows for JsRuntimes with no loader to correctly resolve pre-loaded modules which I think makes more sense (see `es_snapshot_without_runtime_module_loader`).

Reply:
Seems fine 👍